docs/platforms/flutter/data-management/apple-privacy-manifest.mdx

```diff
@@ -0,0 +1,108 @@
+---
+title: Apple Privacy Manifest
+description: "Troubleshoot and resolve common issues with the Apple Privacy Manifest and Sentry Flutter SDK."
+sidebar_order: 50
+---
+
+<Note>
+This guide requires [@sentry/dart@7.17.0](https://github.com/getsentry/sentry-dart/releases/tag/7.17.0) or newer.
+</Note>
+
+Sentry's SDKs provide error and performance monitoring for mobile applications running on Apple devices. To do this, the SDK needs to access certain information about the device and the application. Some of the APIs required for this are considered privacy-relevant by Apple. In order to submit apps to the App Store, Apple requires all apps - and libraries used within these apps - to provide privacy manifest files stating which APIs are used under which allowed reasons. For more details, read Apple's guidelines on [Describing use of required reason API](https://developer.apple.com/documentation/bundleresources/privacy_manifest_files/describing_use_of_required_reason_api).
+
+<Note>
+The information and steps in this guide are still being worked on and might change because of new tools or updated [Apple requirements](https://developer.apple.com/documentation/bundleresources/privacy_manifest_files).
+</Note>
+
+If you are using dynamically linked frameworks, which is the default for Flutter applications, Apple will automatically process the SDK's privacy manifest. If you are using statically linked libraries, you need to provide the privacy manifest yourself. This guide will help you create the privacy manifest for your Flutter applications.
```

Comment:
this is my understanding from our tests and discussions - please double-check @buenaflor

Reply:
yeah looks good, I think we can keep the guide for the static frameworks just in case